.gitignore

```diff
@@ -7,8 +7,30 @@ default/
 # Mac stuff
 *.DS_Store
 
-# IDE files
+# QT Creator
+*.user
+*.creator.*
+*.autosave
+*.pro.user
+*.qbs.user
 *.qbs.user*
+*.pro.user.*
```

Comment:
I don't see why we would need to ignore pro files again. You're not going to add back a qmake project, are you?